### PR TITLE
Bug 1535179 - add two new workerTypes

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ defaults:
 
   statelessHostname:
     enabled: false
-    secret: process.env.DNS_SERVER_SECRET
+    secret: !env DNS_SERVER_SECRET
```
Comment (petemoore) [inline config.yml]: i'm not familiar with this syntax so getting a review from someone else for this line would be great

File: deploy/etc/worker-types.json [hunk 2]
```diff
@@ -1,5 +1,7 @@
 {
   "hvm-builder": [
+    "android-components-g",
+    "application-services-r",
```
Comment (petemoore) [inline deploy/etc/worker-types.json]: Based on [this report](https://bugzilla.mozilla.org/show_bug.cgi?id=1535352#c0) I suspect we should add some more that just these, but this is a good start for [bug 1535179](https://bugzil.la/1535179), and we can probably add the others in [bug 1535352](https://bugzil.la/1535352).

Comment (djmitche) [reply deploy/etc/worker-types.json]: I think @tomprince is adding a bunch more.

File: config.yml (hunk 1 above)
Comment (djmitche) [reply config.yml]: To be honest, we could just drop this, as nothing ever provides the env var (or if it did, it didn't work).  This is at least less-awfully-broken than the previous :)

**Pull request merged**